modules/subchannel/doc/content/source/postprocessors/SCMDuctHeatPostprocessor.md

```diff
@@ -0,0 +1,19 @@
+# SCMDuctHeatPostprocessor
+
+!syntax description /Postprocessors/SCMDuctHeatPostprocessor
+
+## Overview
+
+!! this comment introduces vertical space
+
+The user needs to specify a subchannel problem. Either a QuadSubChannel1PhaseProblem or a TriSubChannel1PhaseProblem. The postprocessor will calculate the heat $[W]$ that goes in_to/out_of the coolant through the duct, based on the aux variable duct_heat_flux $[W/m^2]$. It will integrate this variable over the duct surface.
```

Comment:
```suggestion
The user needs to specify a subchannel problem. Either a QuadSubChannel1PhaseProblem or a TriSubChannel1PhaseProblem. The postprocessor will calculate the heat flow $[W]$ that goes in_to/out_of the coolant through the duct, based on the aux variable duct_heat_flux $[W/m^2]$. It will integrate this variable over the duct surface.
```

Reply:
make a decision between and into / out of

it determines the sign here